upup/models/cloudup/resources/addons/node-termination-handler.aws/k8s-1.11.yaml.template

```diff
@@ -299,7 +299,6 @@ spec:
         app.kubernetes.io/component: daemonset
         kubernetes.io/os: linux
         k8s-app: aws-node-termination-handler
-        kubernetes.io/os: linux
```

Comment:
This is used in `matchLabels`. I don't think we can remove it.

Reply:
It's duplicate with two lines above.